align.go

```diff
@@ -218,7 +229,7 @@ func (a *Align) export(lines []string) {
 }
 
 // pad s based on the supplied PaddingOpts
-func pad(s string, columnNum, count int, j Justification) string {
+func pad(s string, columnNum, count int, j Justification, sur string) string {
```

Comment:
If the idea of changing to `pad`, this can be changed and be clearer of intent/use.

Reply:
Maybe rename function to `applyPad` or `applyPadding` to disambiguate from the noun for pad.

Reply:
Good call - done.